go-controller/pkg/ovn/master.go

```diff
@@ -811,6 +821,15 @@ func (oc *DefaultNetworkController) addUpdateNodeEvent(node *kapi.Node, nSyncs *
 	return err
 }
 
+func (oc *DefaultNetworkController) addUpdateRemoteNodeEvent(node *kapi.Node) error {
```

Comment:
q: why don't we need a `DeleteRemoteNode` func? is it because when node is removed all the remote ports get removed automatically so nothing to actually delete unlike the add case?

Reply:
When a node is deleted (be it local or remote), the function deleteNodeEvent() will handle it.